excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SpreadsheetValidator.java

```diff
@@ -74,23 +79,49 @@ private void validateFixedListWorkSheetIsPresent(Map<String, DefinitionSheet> de
                           String cellValue, String rowNumberInfo) {
         if (columnNameEnum != null) {
             Integer columnMaxLength = columnNameEnum.getMaxLength();
             if (columnMaxLength != null && cellValue.length() > columnMaxLength) {
                 String invalidImportMessage = getImportValidationFailureMessage(sheetName, columnName,
-                                                                                columnMaxLength, rowNumberInfo);
+                    columnMaxLength, rowNumberInfo);
                 throw new InvalidImportException(invalidImportMessage);
             }
         }
     }
 
     public String getImportValidationFailureMessage(String sheetName, String columnName, Integer columnMaxLength,
-                                     String rowNumberInfo) {
+                                                    String rowNumberInfo) {
         return String.format("Invalid Case Definition sheet - In sheet '%s' the column '%s' value should not" +
-                                      " be more than '%s' characters length %s", sheetName,
-                                      columnName, columnMaxLength, rowNumberInfo);
+                " be more than '%s' characters length %s", sheetName,
+            columnName, columnMaxLength, rowNumberInfo);
+    }
+
+    private void validateDisplayContextParameter(Map<String, DefinitionSheet> definitionSheets) {
+        List<DefinitionSheet> definitionSheetList = new ArrayList<>();
+        definitionSheetList.add(definitionSheets.get(SheetName.COMPLEX_TYPES.getName()));
+        definitionSheetList.add(definitionSheets.get(SheetName.CASE_EVENT_TO_COMPLEX_TYPES.getName()));
+        definitionSheetList.add(definitionSheets.get(SheetName.WORK_BASKET_INPUT_FIELD.getName()));
+        definitionSheetList.add(definitionSheets.get(SheetName.WORK_BASKET_RESULT_FIELDS.getName()));
+        definitionSheetList.add(definitionSheets.get(SheetName.SEARCH_INPUT_FIELD.getName()));
+        definitionSheetList.add(definitionSheets.get(SheetName.SEARCH_RESULT_FIELD.getName()));
+
+        definitionSheetList.forEach(definitionSheet -> {
+            if (definitionSheet != null) {
+                definitionSheet.getDataItems().forEach(item -> {
+                    if (item.getString(ColumnName.DISPLAY_CONTEXT_PARAMETER) != null) {
+                        if (isFieldTypeTableOrList(item.getString(ColumnName.DISPLAY_CONTEXT_PARAMETER))) {
+                            throw new InvalidImportException(item.getString(ColumnName.CASE_FIELD_ID) + " contains incorrect or invalid configuration in tab " + definitionSheet.getName());
+                        }
+                    }
+                });
+            }
+        });
+    }
+
+    private boolean isFieldTypeTableOrList(String displayContextColumn) {
+        return displayContextColumn.startsWith(LIST_PREFIX) || displayContextColumn.startsWith(TABLE_PREFIX);
     }
```

Comment:
we could move this inside the model class `DefinitionSheet`:
 if (item.isDisplayContextParameter(DisplayContextParameter.LIST) || item.isDisplayContextParameter(DisplayContextParameter.TABLE) || ) {

Reply:
it would allow better reusability in the future and make code cleaner